primedev/scripts/client/scriptoriginauth.cpp

```diff
@@ -18,6 +19,18 @@ global struct MasterServerAuthResult
 }
 */
 
+ADD_SQFUNC("void", NSResetToken, "", "", ScriptContext::UI)
+{
+	g_pCVar->FindVar("serverfilter")->SetValue("");
+	return SQRESULT_NULL;
+}
+
+ADD_SQFUNC("bool", NSIsVanilla, "", "", ScriptContext::SERVER | ScriptContext::CLIENT | ScriptContext::UI)
```

Comment:
Unnecessary, use `VANILLA` constant instead.

Reply:
Won't work for uiscript